src/scenes/CreateEvent/index.js

```diff
@@ -378,8 +398,53 @@ export default class CreateEvent extends Component {
     if (nextProps.txReturn) {
       this.onClose();
     }
+
+    if (!_.isUndefined(nextProps.addressValidated)) {
+      this.resultSetterField();
+    }
   }
 
+  resultSetterField() {
+    const { intl: { formatMessage } } = this.props;
+    return (
+      <Field
+        required
+        fullWidth
+        name={ID_RESULT_SETTER}
+        placeholder={formatMessage(messages.resultSetterPlaceholder)}
+        component={this.resultSetterTextField}
+        validate={[this.validateResultSetterAddress]}
+      />
+    );
+  }
+
+  resultSetterTextField = ({ input, placeholder, startAdornmentLabel, meta: { touched, error }, ...custom }) => {
+    const { intl, addressValidated } = this.props;
+    let errorMsg = null;
+    if (touched) {
+      if (error) {
+        errorMsg = error;
+      } else if (!addressValidated) {
+        errorMsg = intl.formatMessage(messages.invalidAddress);
+      }
+    }
+
+    return (
+      <FormControl fullWidth>
+        <_TextField
```

Comment:
this is named `_TextField` so that we can reuse `TextField` like this `<Field component={TextField} />`. Please turn into stateless component and follow that convention :)